pkg/node/manager/manager.go

```diff
@@ -409,7 +403,7 @@ func (m *Manager) NodeUpdated(n nodeTypes.Node) {
 		// through the tunnel to preserve the source identity as part of the
 		// encapsulation. In encryption case we also want to use vxlan device
 		// to create symmetric traffic when sending nodeIP->pod and pod->nodeIP.
-		if address.Type == addressing.NodeCiliumInternalIP || m.conf.EncryptionEnabled() ||
```

Comment:
Upstream this was changed to `m.conf.NodeEncryptionEnabled()`, but not completely dropped. Before I start digging deeper, is there an easy reasoning for that diff?

Reply:
Fixed here and in the v1.12 PR (which is also now ready). I will send a PR to fix it in v1.13 for consistency.

Reply:
Thanks for spotting this!!